docs/events/security_alerts/symbols_collision.md

```diff
@@ -0,0 +1,52 @@
+# symbols_collision
+
+## Intro
+symbols_collision - a shared object loaded to a process has collisions in exported symbols with another loaded shared object.
+
+## Description
+An event marking that a shared object loaded to current process, and have collisions of exported symbols
+with other shared object already loaded to the process. This event can help in inform on the 
+occasion that a shared object tries to override some symbol of another library.
+
+### Configuring the event
+The event is configured using arguments filtering.
```

Comment:
The other symbol events had an example of filtering and what to expect. Shouldn't we have the same here? Okay if you prefer not because of X reasons.